### PR TITLE
Release 2.15.2

### PR DESCRIPTION
**Pull request by kuzmany**
**Please be sure you are submitting this against the _staging_ branch.**

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | 
| New feature? | 
| Automated tests included? |
| Related user documentation PR URL | 
| Related developer documentation PR URL | 
| Issues addressed (#s or URLs) | 
| BC breaks? | 
| Deprecations? | 

[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )

[//]: # ( Required: )
#### Description:

[//]: # ( As applicable: )
#### Steps to reproduce the bug:
1. 
2. 

#### Steps to test this PR:
1. Load up [this PR](https://mautibox.com)
2. 

#### List deprecations along with the new alternative:
1. 
2. 

#### List backwards compatibility breaks:
1. 
2.

### CODE REVIEW

**Comment by escopecz** [comment]
Aha, this is not a issue of incoming changes but on master. We missed it in https://github.com/mautic/mautic/pull/7582

**Comment by escopecz** [comment]
@kuzmany I found it. @heathdutton prepared a fix for Travis and it was merged to staging:

https://github.com/mautic/mautic/pull/7611

That's why staging works and master is falling. Add that PR to 2.15.2 release (this PR) and everything should be OK.

**Comment by kuzmany** [comment]
Now it fails due your  unserialize second parameter (already discussed)  
https://travis-ci.org/mautic/mautic/jobs/566041387

Release it with travis error?

**Comment by Gregy** [comment]
I think we need to. We should also publish a release notes that strongly discourage using mautic with php 5.x because the vulnerability is NOT fixed in php 5.x. Meaning anyone can easily take over any mautic installation running on old php (even when they use the latest mautic)

**Comment by escopecz** [comment]
Let me play with it a little more. Maybe I'll be able to create a helper to encapsulate all unserializing and check the PHP version in it.

**Comment by escopecz** [comment]
Looks like we are green and ready. @kuzmany please make a review of the last commits before merging. I checked the code 3 times but a second pair of eyes may find something I missed.

File: app/bundles/CoreBundle/Helper/Serializer.php [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+namespace Mautic\CoreBundle\Helper;
+
+class Serializer
+{
+    /**
+     * Unserializing a string can be a security vulnerability as it can contain classes that can execute a PHP code.
+     * PHP >=7 has the `['allowed_classes' => false]` option to disable classes altogether or whitelist those needed.
+     * PHP <7 do not accept the second parameter, throw warning and return false so we have to handle it diffenetly.
+     * This helper method is secure by default and handle all PHP versions.
```
Comment (Gregy) [inline app/bundles/CoreBundle/Helper/Serializer.php]: I believe this is misleading. This method is not secure by default in php < 7. We should note that here.

**Comment by Gregy** [comment]
Thank you

**Pull request merged**